src/lib/ui/CommunityDelete.svelte

```diff
@@ -0,0 +1,71 @@
+<script lang="ts">
+	import type {Readable} from 'svelte/store';
+	import Message from '@feltcoop/felt/ui/Message.svelte';
+
+	import {getApp} from '$lib/ui/app';
+	import PersonaAvatar from '$lib/ui/PersonaAvatar.svelte';
+	import CommunityAvatar from '$lib/ui/CommunityAvatar.svelte';
+	import type {Community} from '$lib/vocab/community/community';
+	import type {Persona} from '$lib/vocab/persona/persona';
+
+	const {dispatch} = getApp();
+
+	export let community: Readable<Community>;
+	export let persona: Readable<Persona>;
+	export let done: (() => void) | undefined = undefined;
+
+	let errorMessage: string | undefined;
+	let locked = true;
+	let lockText = '';
+
+	const deleteCommunity = async () => {
+		errorMessage = '';
+		const result = await dispatch.DeleteCommunity({
+			community_id: $community.community_id,
+		});
+		if (result.ok) {
+			done?.();
+		} else {
+			errorMessage = result.message;
+		}
+	};
+
+	const onKeydown = async (e: KeyboardEvent) => {
+		if (e.key === 'Enter') {
```

Comment:
```suggestion
		if (!locked && e.key === 'Enter') {
```

same change is needed in `SpaceDelete.svelte`

Reply:
great catch!